embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/logging/InternalErrorLogger.kt

```diff
@@ -1,15 +1,12 @@
 package io.embrace.android.embracesdk.logging
 
-internal class InternalErrorLogger(
```

Comment:
This was so confusing to me when I first looked at this code. This is NOT a Logger in the sense that InternalEmbraceLogger is a Logger - it's a LoggerAction attached to a Logger, so lets just call it that! Names are important yo.